infra/src/dsl/map-to-values.ts

```diff
@@ -33,6 +33,21 @@ export const serializeService: SerializeMethod = (
   uberChart: UberChartType,
 ) => {
   let allErrors: string[] = []
+  const checkCollisions = (target: any, source: any) => {
```

Comment:
No way to create some type better than any here? Isn't the value always either a string or a secret ref?

Reply:
I thought I would leave it open but this is way too wide, I agree. Will fix.